ThunderCloud/VideoListItem.swift

```diff
@@ -38,6 +38,16 @@ open class VideoListItem: VideoListItemView {
 	override open func handleSelection(of row: Row, at indexPath: IndexPath, in tableView: UITableView) {
 		
 		guard let videoRow = row as? VideoListItem, let videos = videoRow.videos else { return }
-		videoRow.parentNavigationController?.push(videos: videos)
+        
+        guard let video = videos.first(where: { $0.localeString == StormLanguageController.shared.currentLanguage }) ?? videos.first, let link = video.link else {
```

Comment:
Guessing this has been okayed with the clients we've chatted this over with too? :D

Reply:
That's correct yeah, it's been okayed & is already in production for some apps :D